packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -299,39 +304,39 @@ interface ActionsBarProps extends IContainerActions {
 function ActionsBar(props: ActionsBarProps): React.ReactElement {
 	const { isContainerConnected, tryConnect, forceDisconnect, closeContainer } = props;
 
-	const connectButtonTooltipId = useId("connect-button-tooltip");
-	const disconnectButtonTooltipId = useId("disconnect-button-tooltip");
-	const disposeContainerButtonTooltipId = useId("dispose-container-button-tooltip");
+	// const connectButtonTooltipId = useId("connect-button-tooltip");
+	// const disconnectButtonTooltipId = useId("disconnect-button-tooltip");
+	// const disposeContainerButtonTooltipId = useId("dispose-container-button-tooltip");
```

Comment:
Cleanup

Reply:
Thank you for the catch.

Reply:
Actually, can we keep the tooltips? These are an important accessibility feature.

Reply:
I will revert the changes to include tooltips. Thanks for the feedback!